core/db/migrate/20150515211137_fix_adjustment_order_id.rb

```diff
@@ -0,0 +1,66 @@
+class FixAdjustmentOrderId < ActiveRecord::Migration
+  def change
+    say 'Populate order_id from adjustable_id where appropriate'
+    execute(<<-SQL.squish)
```

Comment:
This is likely because you where reviewing while I was still editing.

Now all 3 update statements are constrainted to only hit the orders where `order_id IS NULL`.